addons/skin.estuary/1080i/Defaults.xml

```diff
@@ -151,9 +151,10 @@
 		<aligny>center</aligny>
 		<texturefocus border="40" colordiffuse="button_focus">buttons/button-fo.png</texturefocus>
 		<texturenofocus border="40">buttons/button-nofo.png</texturenofocus>
+		<orientation>horizontal</orientation>
 		<sliderwidth>150</sliderwidth>
 		<sliderheight>28</sliderheight>
```

Comment:
you can't define default values for both.
i would keep it as is and define the width/height values on each individual vertical settings slider you use.